src/components/layout.js

```diff
@@ -22,7 +22,7 @@ const Layout = ({ children }) => {
       <Header />
       <div className="flex-horizontal" id="header">
         <div className="img" id="header-image">
-          <img src="/images/csaheader.png" alt="CSA Header" />
+          <img src="/images/csaheader.jpg" alt="CSA Header" />
```

Comment:
To make this a little prettier we can use `object:-fit: cover` in the CSS for this. 
See line 38 in main.css for the CSS selector & properties for this image!